index.html

```diff
@@ -0,0 +1,252 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <!--  links -->
+    <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;600&family=Poppins&family=Roboto&display=swap"
+    rel="stylesheet"
+    />
+    <link
+     rel="stylesheet"
+     href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+   />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+     <!-- header start-->
+     <header class="header">
+        <!-- menu start-->
+       <nav>
+       <p class="logo" id="logo"> Gulalai Company</p>
+       <div >
+           <span class="fa fa-bars"   aria-hidden="true"></span>
+         </div>
+       <ul class="menu"  >
+           <li><a class="link" href="#">protofolio</a></li>
+           <li><a class="link" href="#">about</a> </li>
+           <li><a class="link" href="#">contact</a></li>
+           <li><a class="link " href="#"><i class="fa fa-envelope-o"></i></a></li>
+       </ul>
+        
+       </nav> 
```

Comment:
- In your mobile version, your navbar doesn't exactly match as it is in the Figma template. There should be equal margin both on the ;left and right. Please look into fixing this issue 👍 👍,remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project( Please pay attention to the margin on the right side of the menu bar):
<img width="285" alt="navbar student" src="https://user-images.githubusercontent.com/73607512/194550949-5f8c6e5f-b81f-472e-b760-38eca09387fa.PNG">

Figma template:
<img width="138" alt="nabvar figma 2" src="https://user-images.githubusercontent.com/73607512/194551036-6a5ef6e6-6afd-4465-8229-fd15460a54e1.PNG">

Reply:
Fixed.